app/navigation/NavigatorForChangeActivity.scala

```diff
@@ -113,4 +112,22 @@ class NavigatorForChangeActivity @Inject()() extends Navigator {
     case HowManyOperatePackagingSiteOwnBrandsPage => _ => routes.ChangeActivityCYAController.onPageLoad
     case _ => _ => routes.ChangeActivityCYAController.onPageLoad
   }
+
+  private def navigationFollowingImports(userAnswers: UserAnswers): Call = {
+    userAnswers.get(AmountProducedPage) match {
+      case Some(AmountProduced.Large) => navigateForLargeAmountProducedFollowingImports(userAnswers)
+      case _ => defaultCall
+    }
+  }
+
+  private def navigateForLargeAmountProducedFollowingImports(userAnswers: UserAnswers): Call =
+    (userAnswers.get(OperatePackagingSiteOwnBrandsPage), userAnswers.get(ContractPackingPage)) match {
+      case (Some(opsob), Some(cp)) if (opsob || cp) && userAnswers.packagingSiteList.isEmpty =>
```

Comment:
do we need to wipe this answer when they never see it in the other cases below?

Reply:
I think that is a bit of an overkill. The user can clearly have packaging sites even if they don't meet the criteria where they must have them. We never remove there current packaging sites unless they explicit remove them and the user themselves wouldn't know they was asked about packaging sites because of the answer they gave a few questions back.